include/tester/ssp/SSPAccount.hpp

```diff
@@ -1,44 +1,44 @@
 #pragma once
 
 #include<iostream>
+#include<string>
+
 #include<pjsua2.hpp>
 
 #include"SSPCall.hpp"
 
 class SSPAccount : public pj::Account
 {
 public:
-    SSPAccount() {}
-    ~SSPAccount() 
+    SSPAccount(const std::string & id, const std::string & domain, const std::string & secret,
+        std::function<void(const pj::OnIncomingCallParam &iprm)> onIncomingCall):
+        _onIncomingCall(onIncomingCall)
     {
-        shutdown();
-        std::cout << "*** Account is being deleted: No of _calls="
-            << _calls.size() << std::endl;
-
-        for (std::vector< pj::Call *>::iterator it = _calls.begin();
-                it != _calls.end(); )
-            {
-                delete (*it);
-                it = _calls.erase(it);
-            }
+        std::string uri = SIP + id + "@" + domain;
```

Comment:
No magic strings - use a constexpr here as well

Reply:
How do you want me to call the member?

Reply:
`SEPARATOR` should be fine

Reply:
done